uaclient/conftest.py

```diff
@@ -144,6 +154,10 @@ def __init__(
         ) -> None:
             if not cfg_overrides.get("data_dir"):
                 cfg_overrides.update({"data_dir": tmpdir.strpath})
+            if not cfg_overrides.get("log_file"):
+                cfg_overrides.update(
+                    {"log_file": tmpdir.join("log_file.log").strpath}
+                )
```

Comment:
I think this part is supposed to be in the second commit

Reply:
Updated @orndorffgrant .

Reply:
Thanks!